amazon/ion/writer_binary_raw.py

```diff
@@ -0,0 +1,365 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at:
+#
+#    http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Writer for raw binary Ion values, without symbol table management."""
+
+# Python 2/3 compatibility
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import struct
+from decimal import Decimal
+from datetime import datetime
+from functools import partial
+
+import six
+
+from .writer_binary_raw_fields import _write_varuint, _write_uint, _write_varint, _write_int
+from .core import IonEventType, IonType, DataEvent, Transition
+from .util import coroutine, Enum
+from .writer import writer_trampoline, partial_transition, WriteEventType
+
+
+class _TypeIds(Enum):
```

Comment:
We should probably consolidate this with the ones in the `binary_reader` at some point.  Added #14 to track this.